src/zenml/artifact_stores/base_artifact_store.py

```diff
@@ -211,18 +214,20 @@ class MyArtifactStoreConfig(BaseArtifactStoreConfig):
                     """
                 )
             )
-        values["path"] = values["path"].strip("'\"`")
-        if not any(
-            values["path"].startswith(i) for i in cls.SUPPORTED_SCHEMES
-        ):
-            raise ArtifactStoreInterfaceError(
-                f"The path: '{values['path']}' you defined for your "
-                f"artifact store is not supported by the implementation of "
-                f"{cls.schema()['title']}, because it does not start with "
-                f"one of its supported schemes: {cls.SUPPORTED_SCHEMES}."
-            )
 
-        return values
+        if "path" in data:
```

Comment:
Before `path` was always in `values`, why this is not the case anymore?

Reply:
Maybe I can answer both this one and the question above in a single answer. I chose to use a `before` validator because we were playing around with the incoming values and it felt a bit more comfortable to use a `before` validator. Functionally, it does not change much in this case though.

The `if` condition was added due to linting. Since this is a before validator it may or may not have the key `path`. However, if you check the model description itself, `path` is a required value. Without a `path`, you will not get past validation in any case.